gokart/utils.py

```diff
@@ -76,4 +78,5 @@ def load_dill_with_pandas_backward_compatibility(file: FileLike) -> Any:
     try:
         return pd.read_pickle(file)
     except Exception:
+        file.seek(0)
```

Comment:
https://github.com/search?q=repo%3Aspotify%2Fluigi+seekable&type=code 
Some processors are not seekable. 
Plz check `file.seekable()` here and commenting limitations.

Reply:
@Hi-king 
I've added seeable assertion. Due to https://github.com/m3dev/gokart/pull/382/files#diff-d30054885311f8eb77523063db114b013d94dd98d39a2d4e34a6f3ec5f461bb1R85 , non-seekable file should not come here but I've added assertion in case.

Reply:
@mski-iksm Perfect! THX!